app/components/waive-hearings-popup.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+import { action } from '@ember/object';
+
+// set property in each object of the array to same value
+export function updateEachObjectInArray(records = [], prop, newVal) {
+  records.forEach(function(obj) {
+    obj.set(prop, newVal);
+  });
+}
+
+export default class WaiveHearingsPopupComponent extends Component {
+  showPopup = false;
+
+  // run function updateEachObjectInArray on dispositions array
+  // occurs when a user chooses to opt out of hearings for their project
+  @action
+  onConfirmOptOutHearing(dispositions) {
+    updateEachObjectInArray(dispositions, 'dcpPublichearinglocation', 'waived');
```

Comment:
You _may_ be able to use a method like this for setting values on each prop of an array: https://api.emberjs.com/ember/3.13/classes/EmberArray/methods/setEach?anchor=setEach

Reply:
cool I'll check it out :)